src/index.tsx

```diff
@@ -188,7 +187,9 @@ const BugsnagErrorBoundary = BUGSNAG_API_KEY
   : React.Fragment;
 
 if (MIXPANEL_PROJECT_TOKEN) {
-  mixpanel.init(MIXPANEL_PROJECT_TOKEN, { batch_requests: true });
+  window.mixpanel.init(MIXPANEL_PROJECT_TOKEN, {
+    batch_requests: true,
+  });
```

Comment:
I'd move all the logic related to mixpanel to a separate service. And in the places where it's needed, I'd call it something like `initMixpanel`, `trackVenueEntrance(...params)`, etc.
so that I could easily change the logic in the functions without finding all the places where the mixpanel was used.

Reply:
fixed in 45c8facc226c9f6cd04226cd497c5d3662ab2527